src/components/icons/ExtensionMultiSigIcon.tsx

```diff
@@ -5,7 +5,8 @@ const SVG = (
   <>
     <circle cx="128" cy="128" r="126" fill="#039855" />
     <path
-      d="M25.9375 20.4375H11.488C11.7166 19.9872 11.946 19.524 12.1755 19.053C13.4585 19.1948 14.9787 18.2968 16.7078 16.3701C16.7594 16.5127 16.8169 16.6597 16.8797 16.8084C17.4391 18.1189 18.161 18.8657 19.0281 19.0298C19.9167 19.2017 20.7967 18.7523 21.6991 17.6703C22.2851 18.3638 23.4883 19.0625 25.9375 19.0625C26.1198 19.0625 26.2947 18.9901 26.4236 18.8611C26.5526 18.7322 26.625 18.5573 26.625 18.375C26.625 18.1927 26.5526 18.0178 26.4236 17.8889C26.2947 17.7599 26.1198 17.6875 25.9375 17.6875C23.3052 17.6875 22.5387 16.7774 22.5 16.2936C22.4958 16.1337 22.436 15.9802 22.3308 15.8596C22.2256 15.7391 22.0817 15.659 21.9238 15.6331C21.766 15.6073 21.604 15.6373 21.4659 15.718C21.3277 15.7987 21.2221 15.9251 21.1671 16.0753C20.1212 17.6437 19.5016 17.7202 19.2765 17.6789C18.5606 17.544 17.8946 15.8802 17.6703 14.7991C17.641 14.6626 17.5707 14.5382 17.4689 14.4426C17.3672 14.347 17.2387 14.2846 17.1006 14.2638C16.9625 14.243 16.8213 14.2647 16.6959 14.3261C16.5705 14.3875 16.4667 14.4857 16.3984 14.6075C14.6969 16.7233 13.5556 17.4434 12.8398 17.6273C13.5694 16.0048 14.1142 14.5809 14.4631 13.382C15.0492 11.3677 15.0922 9.95914 14.5929 9.07656C14.3351 8.61594 13.7945 8.0668 12.71 8.0625H12.6885C11.3135 8.07195 10.2307 9.37734 9.71335 11.6461C9.40483 12.9962 9.35413 14.4984 9.57327 15.7711C9.79241 17.0438 10.2487 17.9634 10.9096 18.5116C10.5899 19.1759 10.2625 19.8248 9.94452 20.4392H8.06249C7.88015 20.4392 7.70528 20.5117 7.57635 20.6406C7.44742 20.7695 7.37499 20.9444 7.37499 21.1267C7.37499 21.3091 7.44742 21.4839 7.57635 21.6129C7.70528 21.7418 7.88015 21.8142 8.06249 21.8142H9.21491C8.2421 23.6189 7.48585 24.877 7.47382 24.8977C7.4248 24.9751 7.39176 25.0615 7.37663 25.1519C7.3615 25.2423 7.36458 25.3347 7.3857 25.4239C7.40682 25.5131 7.44556 25.5971 7.49962 25.6711C7.55369 25.7451 7.62199 25.8075 7.70053 25.8547C7.77906 25.9019 7.86624 25.9329 7.95694 25.9459C8.04764 25.959 8.14003 25.9537 8.22867 25.9305C8.31732 25.9073 8.40043 25.8667 8.47314 25.8109C8.54584 25.7551 8.60666 25.6854 8.65202 25.6058C8.66491 25.5834 9.62054 23.991 10.7747 21.8142H25.9375C26.1198 21.8142 26.2947 21.7418 26.4236 21.6129C26.5526 21.4839 26.625 21.3091 26.625 21.1267C26.625 20.9444 26.5526 20.7695 26.4236 20.6406C26.2947 20.5117 26.1198 20.4392 25.9375 20.4392V20.4375ZM22.5 16.3125V16.2945C22.5006 16.3005 22.5006 16.3065 22.5 16.3125ZM11.0523 11.952C11.3951 10.4498 12.0569 9.4375 12.7031 9.4375C13.1543 9.4375 13.3047 9.59734 13.3906 9.75289C13.6484 10.2109 13.9509 11.8317 11.5301 17.1753C10.8262 16.2051 10.5865 13.9922 11.0523 11.952Z"
+      transform="scale(7.5)"
```

Comment:
Not really a fan of "out of our code control" transformations, but I see than in the case of this icon, due to it was built it won't work without.

We could go back and try to re-create it in order to avoid this transformation, but I honestly don't think the time investment is worth it.

Reply:
Yes, I was thinking about the same, to re-create the icon, given the original icon had this viewBox="0 0 35 34", but the IconBase has a much larger one, so the path is not scaling properly, but this seemed the much faster solution